pavelib/assets.py

```diff
@@ -324,6 +324,10 @@ def update_assets(args):
         help="Skip collection of static assets",
     )
     parser.add_argument(
+        '--skip-coffee', dest='coffee', action='store_false', default=True,
```

Comment:
nit: could you rename `dest='coffee'` to be more-obviously about not-using coffee (e.g., `dest='skip_coffee'`)?